<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTreeWidget</class>
 <widget class="QWidget" name="DataTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolbar">
     <item>
      <widget class="QLineEdit" name="searchInput">
       <property name="placeholderText">
        <string>æœç´¢æ•°æ®ç‚¹...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expandBtn">
       <property name="text">
        <string>å…¨éƒ¨å±•å¼€</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="collapseBtn">
       <property name="text">
        <string>å…¨éƒ¨æŠ˜å </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>åˆ·æ–°</string>
       </property>
       <property name="text">
        <string>ğŸ”„</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>åç§°</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>å€¼</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ç±»å‹</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>æè¿°</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="styleSheet">
      <string notr="true">color: #666;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
